### PR TITLE
Removed txpool add command

### PR DESCRIPTION
**Pull request by ZeljkoBenovic**
# Description

This PR removes the ```txpool add``` cli command as we do not support unsigned transactions.

# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [ ] I have added the relevant labels
- [x] I have updated the official documentation
- [x] I have added sufficient documentation in code

## Testing

- [ ] I have tested this code with the official test suite
- [x] I have tested this code manually

### Manual tests

```go run main.go txpool --help``` should not show the add subcommand

# Documentation update

https://github.com/0xPolygon/polygon-edge-docs/pull/73

### CODE REVIEW

File: command/txpool/txpool.go [hunk 1]
```diff
@@ -23,8 +22,8 @@ func GetCommand() *cobra.Command {
 
 func registerSubcommands(baseCmd *cobra.Command) {
 	baseCmd.AddCommand(
-		// txpool add
-		add.GetCommand(),
+		// txpool add - removed
+		// we don't support adding unsigned transactions
```
Comment (zivkovicmilos) [inline command/txpool/txpool.go]: You can completely remove these two lines

**Review by zivkovicmilos — APPROVED** [review]
Looks good 💯

**Review by Kourin1996 — APPROVED** [review]
LGTM apart from Milos' comment

File: command/txpool/txpool.go (hunk 1 above)
Comment (ZeljkoBenovic) [reply command/txpool/txpool.go]: Removed in 44aa6d3b2da32c187d9100d5abda802d9164d8ae

**Review by 0xAleksaOpacic — APPROVED** [review]

**Pull request merged**